<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>237</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Synshot settings</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>230</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>40</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>40</number>
    </property>
    <item row="0" column="1">
     <widget class="QLineEdit" name="defaultSSHPort">
      <property name="inputMask">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="defaultPortLabel_2">
      <property name="text">
       <string>Notification sound</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="defaultPortLabel_3">
      <property name="text">
       <string>Screenshot dir</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="defaultPortLabel_4">
      <property name="text">
       <string>Destination remote dir</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="defaultPortLabel">
      <property name="text">
       <string>Default SSH port</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="screenshotDir">
      <property name="inputMask">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="notificationSound">
      <property name="inputMask">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="defaultPortLabel_5">
      <property name="text">
       <string>Remote URL path</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="destinationRemoteDir">
      <property name="inputMask">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="urlPrefix">
      <property name="inputMask">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Synshot.qrc"/>
 </resources>
 <connections/>
</ui>
